iot-hub/Samples/service/CleanupDevicesSample/CleanupDevicesSample.cs

```diff
@@ -65,8 +69,29 @@ private async Task CleanupDevices()
                     excludeKeysInExport: true,
                     storageAuthenticationType: StorageAuthenticationType.KeyBased);
 
-            JobProperties exportAllDevicesJob = await _registryManager.ExportDevicesAsync(exportAllDevicesProperties);
+            JobProperties exportAllDevicesJob = null;
+            
+            int tryCount = 0;
+            while (true)
+            {
+                try
+                {
+                    exportAllDevicesJob = await _registryManager.ExportDevicesAsync(exportAllDevicesProperties);

```

Comment:
```suggestion
                    exportAllDevicesJob = await _registryManager.ExportDevicesAsync(exportAllDevicesProperties).ConfigureAwait(false);

```

Reply:
From what I know, we don't use configureawait in samples.

Reply:
You are right. We do have `ConfigureAwait()` elsewhere in this file, though, so we should clean that up then.

Reply:
Will clean up!